Source/GTMSessionFetcher.m

```diff
@@ -223,8 +223,9 @@ @implementation GTMSessionFetcher {
   id _userData;            // retained, if set by caller
   NSMutableDictionary *_properties;  // more data retained for caller
   dispatch_queue_t _callbackQueue;
-  dispatch_group_t _callbackGroup;   // read-only after creation
-  NSOperationQueue *_delegateQueue;  // immutable after beginFetch
+  dispatch_queue_t _underlyingCallbackQueue;  // the target of the serial _callbackQueue
```

Comment:
Intermediate trolling

Reply:
Done!

I put the read-only property in `GTMSessionUploadFetcher.m` similar to the other protected properties/methods. I also updated `GTMSessionUploadFetcher` to use `serialCallbackQueue` instead of `callbackQueue`.